### DISCUSSION TITLE
Welcome to Hacktoberfest-2022 Discussions!

### DISCUSSION THREAD

**Discussion by SyedZawwarAhmed** [SyedZawwarAhmed/Hacktoberfest-2023#63] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ukesh-dev** [comment]
Hi! I am a frontend developer. I want to connect & collaborate with as many people as possible

**Reply by SyedZawwarAhmed** [reply]
Hello there, yeah sure, you can connect with me on github as well as other social media platforms, you can find the links in my github README.
